dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -140,30 +342,163 @@ export default defineComponent({
     };
   },
   name: "CreateEUTaxonomyForFinancials",
-  components: { DataPointFormElement, FailedUpload, FormKit, SuccessUpload, Card },
+  components: {
+    JumpLinksSection,
+    FailedUpload,
+    FormKit,
+    SuccessUpload,
+    UploadFormHeader,
+    Card,
+
+    UploadReports,
+    BasicInformationFields,
+
+    PrimeButton,
+    Calendar,
+    MultiSelect,
+    KPIfieldSet,
+  },
 
-  data: () => ({
-    innerClass: {
-      "formkit-inner": false,
-      "p-inputwrapper": true,
+  data() {
+    return {
+      formInputsModel: {} as CompanyAssociatedDataEuTaxonomyDataForFinancials,
+      filesToUpload: [] as ExtendedFile[],
+      uploadFiles: [] as ExtendedCompanyReport[],
+      fiscalYearEnd: "" as Date | "",
+      convertedFiscalYearEnd: "",
+      reportingPeriod: new Date(),
+      assuranceData: [
+        { label: "None", value: "None" },
+        { label: "LimitedAssurance", value: "LimitedAssurance" },
+        { label: "ReasonableAssurance", value: "ReasonableAssurance" },
+      ],
+      maxFileSize: UPLOAD_MAX_FILE_SIZE_IN_BYTES,
+      euTaxonomyKPIsModel: euTaxonomyPseudoModelAndMappings,
```

Comment:
We could/should just use `euTaxonomyPseudoModelAndMappings`, without re-assigning it to the name `euTaxonomyKPIsModel`.

This is confused me a little while reading the code, because it created the impression that the values behind it can actually change.
But in reality it is a constant pseudo model that you are using,  similar to `euTaxonomyKpiNameMappings` and `euTaxonomyKpiInfoMappings`.

Reply:
Or are you forced to do this because you need it in the DOM?

Reply:
ok so there is no name euTaxonomyKPIsModel any longer